### ISSUE TITLE
Linux installed without my consent

### ISSUE THREAD

**Issue by TempAccountToComplain** [builtbybel/BloatyNosy#16]
Thanks for the debloater, but now I have Linux listed in File Explorer.
![empty linux folder](https://user-images.githubusercontent.com/126802142/222482716-653630cb-ca6a-428a-b1b5-3c8da7e51f68.jpg)

When trying to open Properties to see details on it, I get an error "The properties for this item are not available".
![error from Properties](https://user-images.githubusercontent.com/126802142/222482768-499e8ce8-c8ad-4a95-9a47-fa4515c34aea.jpg)

I did not consent to this.  So...  a debloater added bloat to my PC?

How do I remove the whatever iteration of Linux installed from running BloatyNosy?  It is not listed in partitions.  It is not listed in hardware devices.  It is not listed in installed programs.  It is not listed in Features.  There doesn't seem to be a simple way to remove what I didn't ask for.

(yes...  I created an account JUST to complain about this...)

**Comment by Belim** [comment]
You've enabled "Microsoft Windows Subsystem for Linux" feature in BloatyNosy. Many find it useful, which is why it has made its way into BloatyNosy. You can easily disable it by checking the box next to "Feature Recommendation" and then restoring the default settings (button with downward arrow next to the analysis button).

**Comment by TempAccountToComplain** [comment]
That...  worked?
I think it worked.  But I had restored defaults...  so I rebloated Windows?

Anyhow...  the Linux drive/folder/subsystem//icon/whatever is gone.  Luckily, this PC isn't important enough to bother further.

I recommend streamlining the GUI so it's simple to follow.  Having a feature listed at the bottom of a list (accessible via a small-print link-type sentence at the bottom of an oversized window with too much blank space) and describing the feature as to making a user unsure if I should check or uncheck the box to activate or deactivate the feature makes it confusing.

This is one of the reasons I regularly have trouble with open-source software and the like.  I enjoy them, but I have to filter through lots of badly interfaced choices until I find a good one.


Thanks, though!!!
